var/spack/repos/builtin/packages/py-minis-validation/package.py

```diff
@@ -13,23 +13,22 @@ class PyMinisValidation(PythonPackage):
     git      = "ssh://bbpcode.epfl.ch/nse/minis-validation"
 
     version('develop', branch='master')
-    version('0.0.3', tag='minis-validation-v0.0.3')
-    version('0.0.2', tag='minis-validation-v0.0.2')
+    version('0.0.4', tag='minis-validation-v0.0.4')
 
     depends_on('py-setuptools', type=('build', 'run'))
 
-    depends_on('py-numpy@1.14:', type='run')
-    depends_on('py-pandas@0.25:', type='run')
+    depends_on('py-numpy@1.14:1.999', type='run')
+    depends_on('py-pandas@0.25:1.999', type='run')
```

Comment:
Is Pandas 2 going to break things?

Reply:
I'm not sure, because I didn't work on the package, but I enforced the same constraints indicated in `setup.py` for consistency. Maybe it can be discussed outside of this PR and @asanin-epfl can add more comments about that.

Reply:
In this case the constraint is for pandas and numpy, not dask (that's the one usually discussed 😄 ).